docs/src/appendix/cheatcodes/load.md

```diff
@@ -0,0 +1,61 @@
+# `load`
+
+> `fn load(target: ContractAddress, storage_address: felt252, size: felt252) -> Array<felt252> `
+
+Loads a `Array` of felts from `target` contract's storage, starting at `storage_address`.
+
+- `target` - Address of the contract, which storage you want to modify
+- `storage_address` - Offset of the data in the contract's storage 
+- `size` - How many felts will be loaded into the result `Array` 
+
+
+Those examples present the usage in practise:
+
+```rust
+mod Contract {
+    #[storage]
+    struct Storage {
+        plain_felt: felt252,
+        mapping: LegacyMap<felt252, felt252>,
+    }
+}
+
+// ...
+use snforge_std::{ store, load, map_entry_address };
+
+#[test]
+fn load_plain_felt() {
+    let deployed = deploy_contract();
+    let loaded = load(deployed.contract_address, selector!("plain_felt"), 1);
+    assert(loaded.len() == 1, 'Wrong loaded vector');
+    assert(*loaded.at(0) == 0, 'Wrong loaded value');
+}
+
+#[test]
+fn load_map_entry() {
+    let deployed = deploy_contract();
+    load(
+        deployed.contract_address, 
+        map_entry_address(
+            selector!("mapping"), // Providing variable name
+            array![123].span(),   // Providing mapping key 
```

Comment:
```suggestion
            array![123].span(),   // Providing mapping keys 
```
Should it be? Or another example for multiple keys, I feel like it still not clear why it's an array / how to work with nested maps

Reply:
[Add an example how to handle complex mapping key/values in the contract](https://github.com/foundry-rs/starknet-foundry/pull/1525/commits/e7b8185be349c2f96ce5c43e6ac9757672080862)

Reply:
I'm not going to provide example with nested maps since it's highly unusual one, and this example range is already too extensive